spec/features/stories_spec.rb

```diff
@@ -39,6 +39,49 @@
       end
     end
 
+    context 'when the user save a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'highlights story on click to save', js: true do
+        find(story_selector(story)).trigger('click')
+        expect(page).to have_css('.editing')
+        click_on 'Save'
+        expect(page).not_to have_css('.editing')
+        expect(find(story_selector(story))[:style]).to match(/background-color/)
+      end
+    end
+
+    context 'when the user drags a story' do
+      let!(:story) { create(:story, title: 'Story', project: project, requested_by: user) }
+
+      before do
+        visit project_path(project)
+        wait_spinner
+        wait_page_load
+      end
+
+      it 'check if the story is in the correct column', js: true do
+        expect(find('#in_progress')).to have_css("div#{story_selector(story)}")
+        expect(find('#chilly_bin')).not_to have_css("div#{story_selector(story)}")
+      end
+
+
+      it 'drags a story to other column', js: true do
+        source = page.find(story_selector(story))
+        target = page.find('#chilly_bin')
```

Comment:
Note for a future refactor (/cc @andrerpbts), we could have test helpers line `chilly_bin_column` and `story_element(story)` to avoid having to use the selector for those in every test. Doing this way would make us avoid having to change a lot of things if we decide to change those selectors sometime.